typescript/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cooklang/cooklang-ts",
+  "version": "0.0.0",
+  "private": "true",
```

Comment:
Intentionally left this private until we are ready to move forward with this setup.